File: code_samples/python/v1.0.0/chargeback.py
from tatrapayplus import TBPlusSDK
from tatrapayplus.models import *

client = TBPlusSDK(
    "your-client-id",
    "your-client-secret"
)
payment_id = "b54afd37-5bb9-4080-9416-5ec450779087"
chargeback_data = CardPayUpdateInstruction(
    operation_type=CardPayUpdateInstructionOperationType.CHARGEBACK,
    amount=120,
)
chargeback_response = client.update_payment(payment_id, chargeback_data)

File: code_samples/php/v1.0.0/chargeback.php
<?php
use Tatrapayplus\TatrapayplusApiClient\Api\TatraPayPlusAPIApi;
use Tatrapayplus\TatrapayplusApiClient\Model\CardPayUpdateInstruction;

$tatrapayplus_api = new TatraPayPlusAPIApi(
    "your-client-id",
    "your-client-secret",
);
$payment_id = 'b54afd37-5bb9-4080-9416-5ec450779087'; // Retrieved from initiatePayment
$chargeback_data = new CardPayUpdateInstruction([
    "operation_type" => CardPayUpdateInstruction::OPERATION_TYPE_CHARGEBACK,
    "amount" => 3.0,
]);

$response = $tatrapayplus_api->updatePaymentIntent($payment_id, $chargeback_data);